Communitheme/gtk-3.0/_drawing.scss

```diff
@@ -238,6 +238,7 @@
     @if $flat == true { $c: darken($c, 10%); }
     $_bg: if(lightness($c)<35%, lighten($c, 25%), lighten($c, 5%));
     background-color: $_bg;
+    @if $variant=='dark' and $c != darken($headerbar_bg_color, 10%) { background-color: lighten($c,1%); }
```

Comment:
This is because on flat==true, $c is equal to darken($headerbar_bg_color, 10%), right?
What do you think about creating a (e.g.) 
_c_flat_headerbar: darken($hb..., 10%)

and then use the this variable in both cases? It would be clearer that the two values are linked even in the future

Reply:
Great idea. I'll try that